components/dashboard/src/App.tsx

```diff
@@ -71,6 +71,12 @@ function App() {
         }
     }, [localStorage.theme]);
 
+    if (window.location.pathname === '/' && window.location.hash === '' && !hasLoggedInBefore()) {
```

Comment:
Let's remove the `hasLoggedInBefore` so that you are landing on the website when you've logged out.

Reply:
I just learned about the `markLoggedIn` which sets a HTTP cookie `gitpod-user=loggedIn`. 

But we already have a session cookie `gitpod-user=loggedIn`, which is managed on login/logout.

Reply:
`hasLoggedInBefore` checks for a Cookie which is never removed, `gitpod-user=loggedIn` used to reflect the session state. 

WDYT of renaming the new Cookie, so it reflects that particular state? maybe something like "gitpod-logged-in-before". Maybe we can even set the provider host as value ;-)

Reply:
The cookie should be called 'seen-gitpod' or so, as it just indicates the user knows gitpod (has seen the website or the app). Anyhow, we don't want to use it here, as the login state is enough. We really should redirect everyone to www once they are logged out.

Reply:
> Anyhow, we don't want to use it here, as the login state is enough. We really should redirect everyone to www once they are logged out.

Roger that. That thing is, that the "login state" is fetched asynchronously. So, removing of the `gitpod-user=loggedIn` (session) cookie check, would mean that the redirect might happen in any case.

Reply:
This is what used to work before: https://github.com/gitpod-io/gitpod/blob/24f7b609bf79b247d72d1841282639726bcd5891/components/dashboard/src/components/create/index.tsx#L56